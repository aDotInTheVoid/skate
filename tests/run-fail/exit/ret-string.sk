fn main() {
    // TODO: Test case when we return 1;
    return "hello";
}