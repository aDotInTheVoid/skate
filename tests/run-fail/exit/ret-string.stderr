error: `main` returned `String("hello")` with type `string`, expected `null` or `int`

